bin/main

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+there_is_a_winner = false
+puts ''
+puts 'Welcome to Tic-Tac-Toe!'
+puts ''
+puts 'Enter the name for player 1: '
+name_player1 = gets.chomp
+puts 'Enter the name for player 2: '
+name_player2 = gets.chomp
+puts ''
+puts ''
+
+puts "#{name_player1} will have the X"
+puts "#{name_player2} will have the O"
+puts ''
+puts 'HERE THE BOARD WILL BE DRAWN'
+puts ''
+puts "#{name_player1} will have the first move"
+puts ''
+round_num = 0
+
+while round_num < 9
```

Comment:
[Suggestion]
It is better not to use a loop based on a sudden number. Cause it reduces the scalability of a task. So I will suggest you not to use it in your game_logic milestone.